docs/docs/download.md

```diff
@@ -20,18 +20,33 @@ Because the app is not signed, you will need to right-click, or hold the <kbd>co
 
 ## Download for Linux
 
-### Ubuntu 18.04 and newer
+### Flatpak
+
+1. [Set up Flatpak](https://www.flatpak.org/setup/)
+
+2. Install xemu from [Flathub](https://flathub.org/apps/details/org.xemu.xemu)
+
+`flatpak install -y org.xemu.xemu`
+
+3. Run xemu
+
+`flatpak run org.xemu.xemu`
+
+Note: Only the `~/.var/app/app.xemu.xemu/data/xemu/xemu` directory can be written by xemu.
+The Hard Disk image has to be placed there, for example at `~/.var/app/app.xemu.xemu/data/xemu/xemu/xbox_hdd.qcow2`.
+
+### Ubuntu PPA
 
 Users running recent versions of Ubuntu (18.04 and newer) are recommended to install through the daily build PPA repository. At the command line, run the following:
 
-```bash
+```sh
 sudo add-apt-repository ppa:mborgerson/xemu
 sudo apt update
 sudo apt install xemu
 ```
 
 You can now launch xemu through your typical launcher, or by typing `xemu` at the command line.
 
-### Other Linux Distributions
+### Build from source
 
-Users running non-Ubuntu distributions are currently recommended to [build from source](building-from-source.md#linux).
+It is also possible to [build from source](building-from-source.md#linux), which works on all Linux distributions.
```

Comment:
We should not claim it works on all other Linux distros (we haven't verified this)

Reply:
Ok. It should not be recommend then, just mentioning it is possible.